Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/PressableButtons/PressableButton.cs

```diff
@@ -250,15 +250,19 @@ void IMixedRealityTouchHandler.OnTouchStarted(HandTrackingInputEventData eventDa
                     }
                 }
             }
+
+            eventData.Use();
```

Comment:
I'm really not sure what the point of eventData.Use() is since any handled event stops bubbling.

Reply:
At least for global event handlers, this leads to not propagating the event to the pointers' focused objects. This logic is in `MixedRealityInputSystem.HandleEvent`.

Reply:
But I would really be interested what Unity's use of this property is.

Another example: On components on the same game object all events are executed, no matter whether `used` is set or not.

Reply:
Ah okay, so there is a global event handler implication here.

> Another example: On components on the same game object all events are executed, no matter whether used is set or not.

Exactly. Not helpful.